careers/careers/templates/careers/home.jinja

```diff
@@ -133,14 +133,18 @@
   </div>
 </div>
 
-{% if blog_posts %}
-  <section class="mzp-l-card-half">
-  {% for post in blog_posts %}
+{% if recent_posts %}
+  <section class="mzp-l-card-third">
+  {% for post in recent_posts %}
     <div class="mzp-c-card mzp-c-card-medium mzp-has-aspect-16-9">
       <a href="{{ post.link }}" class="mzp-c-card-block-link">
         <div class="mzp-c-card-media-wrapper">
           <div class="lazy-image-container">
+          {% if post.image %}
             <img src="{{ post.image }}" alt="" class="mzp-c-card-image" />
+          {% else %}
+            <img src="{{ static('img/blog-fallback.png') }}" class="mzp-c-card-image" />
```

Comment:
I guess we could use `|default` but maybe it's less readable. Up to you

Reply:
hm, no, i like your suggestion better. will update. 😄